config/config.yaml

```diff
@@ -49,15 +49,23 @@ global:
   # Leave unset for default of launching local browser via geckodriver etc.
   # selenium_url: http://example.com:4444/wd/hub
 
+  # Each key:value pair from this section is exported as shell environment variable
+  # into the running context:
+  # install-envvars:
+  #   foo: "bar"
+
+  # Examples of usage YAML Helper classes
   # Any value you can set to refer to an environment variable
+  #
   # read value from ENVVAR1 environment variable:
-  # test1: !ruby/object:BushSlicer::FlexyEnvVariable
+  # input_entry: !ruby/object:BushSlicer::FlexyEnvVariable
```

Comment:
As a last minute change, may I suggest renaming `FlexyEnvVariable` to `ConfigEnvVariable` (same with `...File`)? It should be clearer this way as well make more sense for the whole verification-tests repo.

Reply:
If you don't want, don't take it as a blocking request.

Reply:
This improvement should be an additional effort which is out of scope of this PR (tracked as ONEOCPDEPL-93).